packages/controls/src/widget_int.ts

```diff
@@ -371,6 +380,40 @@ class IntRangeSliderView extends BaseIntSliderView {
      */
     handleSliderChanged(e, ui) {
```

Comment:
for when continuous_update is set to true, I think I may want to also add validation in handleSliderChange (right now, with this current logic, sliding is unrestricted in continuous mode, but will 'snap back' to the min_gap once you release the mouse- that might not be the most intuitive approach?)